src/test/ui/const-generics/infer/cannot-infer-const-args.stderr

```diff
@@ -2,12 +2,12 @@ error[E0282]: type annotations needed
   --> $DIR/cannot-infer-const-args.rs:6:5
    |
 LL |     foo();
-   |     ^^^ cannot infer the value of const parameter `X` declared on the function `foo`
+   |     ^^^ cannot infer the value of the const parameter `X` declared on the function `foo`
    |
-help: consider specifying the const argument
+help: consider specifying the generic argument
    |
 LL |     foo::<X>();
-   |     ~~~~~~~~
+   |        +++++
```

Comment:
I'm realizing that this message should mention what type `X` is and/or point at the `foo` definition.

Reply:
added to #94483